### ISSUE TITLE
[BUG]: SLPS-25113 Zettai Zetsumei Toshi Graphic Glitch

### ISSUE THREAD

**Issue by nassau-tk** [PCSX2/pcsx2#12005]
### Describe the Bug

Any H/W Rendering will not draw the menu graphics correctly.

[S/W rendering]
![Image](https://github.com/user-attachments/assets/9bdc7d34-fc53-490b-a5a5-a9120c460ef2)

[]H/W rendering]
![Image](https://github.com/user-attachments/assets/f2953770-b017-47c0-9063-2db38e50eebb)

### Reproduction Steps

Choose the H/W rendering any OK.
And, start tutorial.
The Menu will not appear on the screen.

### Expected Behavior

We wish the draw the menu correctly.

### PCSX2 Revision

v2.3.14

### Operating System

Windows 10 (64bit)

### CPU

Ryzen 7 5800X

### GPU

RTX3060

### Logs & Dumps

[Zettai Zetsumei Toshi_SLPS-25113_20241113192105_SWrendering.zip](https://github.com/user-attachments/files/17730777/Zettai.Zetsumei.Toshi_SLPS-25113_20241113192105_SWrendering.zip)
[Zettai Zetsumei Toshi_SLPS-25113_20241113192125_HWrendering.zip](https://github.com/user-attachments/files/17730776/Zettai.Zetsumei.Toshi_SLPS-25113_20241113192125_HWrendering.zip)

**Comment by knight-ryu12** [comment]
Use SW for time being. This is a depth issue and no GPU can do it correctly...

Also, maybe use Discord for further reporting for faster resolve.

**Comment by nassau-tk** [comment]
I have seen on Github for other emulators that display issues related to depth are very troublesome.

I hope it will be resolved someday.

**Comment by Heitof** [comment]
Works here. Don't know how, but the #13805 pr somehow fixed it without adding anything in the gamedb?

<img width="1920" height="1031" alt="Image" src="https://github.com/user-attachments/assets/b5f0c9fb-c625-478d-a960-8156f0a966e6" />

**Comment by refractionpcsx2** [comment]
> Works here. Don't know how, but the [#13805](https://github.com/PCSX2/pcsx2/pull/13805) pr somehow fixed it without adding anything in the gamedb?

It is using that, though

<img width="1359" height="1068" alt="Image" src="https://github.com/user-attachments/assets/9e8555f6-5fcc-4c86-8812-4f2c9650b8c5" />

**Comment by refractionpcsx2** [comment]
it was indeed added in that PR linked

<img width="469" height="263" alt="Image" src="https://github.com/user-attachments/assets/6a9c653f-e3b3-4a41-8ac7-345df58b2612" />

**Comment by Heitof** [comment]
@refractionpcsx2 I was searching for "Zettai" and found nothing on the pr. Turns out I had to click for it to show the games' names. :P..

**Comment by refractionpcsx2** [comment]
yeah it tends to hide the names, too far away from the change. lol
